tests/h/groups/util_test.py

```diff
@@ -25,6 +25,10 @@ def test_pubid(self, group):
 
     def test_is_public(self, group):
         assert group.is_public
+        assert group.pubid == '__world__'
+
+    def test_creator(self, group):
+        assert group.creator == None
```

Comment:
This seems to work, but it should be `assert group.creator is None` to make Hound happy 🐶